src/mount.c

```diff
@@ -309,6 +351,91 @@ cc_handle_mounts(struct cc_oci_config *config, GSList *mounts, gboolean volume)
 }
 
 /*!
+ * Return a rootfs bind mount for container.
+ *
+ * \param config \ref cc_oci_config.
+ *
+ * \return \c cc_oci_mount on success, else \c NULL.
+ */
+struct cc_oci_mount*
+rootfs_bind_mount(struct cc_oci_config *config)
+{
+	struct cc_oci_mount *m;
+
+	if (! config ) {
+		return NULL;
+	}
+
+	m = g_malloc0 (sizeof (struct cc_oci_mount));
+	if (! m) {
+		goto error;
+	}
+
+	m->flags = MS_BIND;
+
+	/* Destination */
+	m->mnt.mnt_dir = g_malloc0(PATH_MAX);
+	if (! m->mnt.mnt_dir) {
+		goto error;
+	}
+
+	g_snprintf(m->mnt.mnt_dir, PATH_MAX, "/%s/rootfs",
+		   config->optarg_container_id);
+
+	/* Source */
+	m->mnt.mnt_fsname = g_strdup(config->oci.root.path);
+	if (! m->mnt.mnt_fsname) {
+		goto error;
+	}
+
+	/* Type */
+	m->mnt.mnt_type = g_strdup("bind");
+	if (! m->mnt.mnt_type) {
+		goto error;
+	}
+
+	g_debug("Rootfs bind mounting done");
```

Comment:
I still think that's a little unclear - that function isn't doing anything with mounts - it's just allocating a struct/object so I'd just log something like:

```
Created rootfs bind mount object
```

I'm not really sure even that is useful though, unless you add some details of the object itself maybe?